bukuserver/static/bukuserver/css/modal.css

```diff
@@ -0,0 +1,31 @@
+/* prevent unnecessary scrollbox from appearing at the main window */
+body.modal-open {
+    overflow-y: inherit;
+}
+
+/* limit dialog height with a scrollbox */
+.modal-content {
+    max-height: calc(100vh - 60px);
```

Comment:
Previous implementation assumed same-sized dialog header, but Bookmarks dialogs are larger; so I moved dynamic size to the dialog frame instead (…which required a few extra hacks to make the scrollbox work correctly).